src/domains/fdr-common/03_cosmos_mongodb_fdr.tf

```diff
@@ -63,105 +63,42 @@ resource "azurerm_cosmosdb_mongo_database" "fdr" {
 locals {
```

Comment:
Maybe there is a missing update of the description
https://github.com/pagopa/pagopa-infra/blob/6e9b7d892ddd8d5cbddc8340d23e648669e9e8d9/src/domains/fdr-common/03_cosmos_mongodb_fdr.tf#L48

Reply:
My bad, just a typo. It will be removed on the next commit.